### PR TITLE
Update 07-data-bindings.adoc

### PR DESCRIPTION
**Pull request by mchacong**
substitutions

### CODE REVIEW

**Review by orlandocr — APPROVED** [review]

File: modules/amp-core/pages/07-data-bindings.adoc [hunk 1]
```diff
@@ -1,9 +1,102 @@
+
 = Using data bindings
 
+The player continues to support very advanced and flexible parameterization. Substitutions in many configuration elements (usually for ads and analytics) continue to use the the syntax `#{variable}` or `${variable}`.
```
Comment (JeanCarlosChavarriaHughes) [inline modules/amp-core/pages/07-data-bindings.adoc]: @mchacong 
typo:
the the

File: modules/amp-core/pages/07-data-bindings.adoc [hunk 2]
```diff
@@ -1,9 +1,102 @@
...
+
 [source,javascript]
 ----
-"${player.mode} Player"
+
+config = {
+    plugins: {
+        analyticsPlugin: {
+            heigth: "${player.videoHeight}", // Current video Heigth 
+            width: "${ads.currentAd.type}", // Current video Width
```
Comment (JeanCarlosChavarriaHughes) [inline modules/amp-core/pages/07-data-bindings.adoc]: @mchacong 
this example seems incorrect,

isn't width: "${player.videoWidth}" 
???

File: modules/amp-core/pages/07-data-bindings.adoc [hunk 3]
```diff
@@ -1,9 +1,102 @@
...
+            ratio: "${player.videoWidth / player.videoHeight}" // The aspect ratio formula is width divided by height
+        }
+    }
+----
+
+See current https://docs.player.akamai.com/static/amp-web-sdk/akamai.amp.HTMLPlayer.html#absoluteCurrentTime[HTMLPlayer] members
+
+=== Customer provided data 
+==== Params (deprecated)
+Prior to v9, the player could hold parameters in a hash table called `params[]`. However, if a lookup occurred and the variable did not exist in `params[]`, the player would look in the DOM for the variable.
```
Comment (JeanCarlosChavarriaHughes) [inline modules/amp-core/pages/07-data-bindings.adoc]: the player would look into

**Review by JeanCarlosChavarriaHughes — COMMENTED** [review]
Please review some minor changes.

File: modules/amp-core/pages/07-data-bindings.adoc (hunk 1 above)
Comment (mchacong) [reply modules/amp-core/pages/07-data-bindings.adoc]: fixed

File: modules/amp-core/pages/07-data-bindings.adoc (hunk 2 above)
Comment (mchacong) [reply modules/amp-core/pages/07-data-bindings.adoc]: yes, fixed

File: modules/amp-core/pages/07-data-bindings.adoc (hunk 3 above)
Comment (mchacong) [reply modules/amp-core/pages/07-data-bindings.adoc]: changed

**Review by JeanCarlosChavarriaHughes — APPROVED** [review]
looks good

**Pull request merged**